contracts/test/vault/oeth-vault.js

```diff
@@ -621,6 +625,20 @@ describe("OETH Vault", function () {
         fixtureWithUser
       );
     });
+    it("Should claim single big request as a whale", async () => {
+      const { oethVault, matt } = fixture;
+
+      await oethVault.connect(matt).requestWithdrawal(oethUnits("30"));
+
+      await advanceTime(delayPeriod); // Advance in time to ensure time delay between request and claim.
+
+      const tx = await oethVault.connect(matt).claimWithdrawal(0); // Claim withdrawal for 50% of the supply
```

Comment:
pls add a post claim check that the OETH supply does not change or the Vault's `totalValue`